controllers/imageupdateautomation_controller.go

```diff
@@ -271,14 +267,11 @@ func durationSinceLastRun(auto *imagev1.ImageUpdateAutomation, now time.Time) ti
 
 // automationsForGitRepo fetches all the automations that refer to a
 // particular source.GitRepository object.
-func (r *ImageUpdateAutomationReconciler) automationsForGitRepo(obj handler.MapObject) []reconcile.Request {
+func (r *ImageUpdateAutomationReconciler) automationsForGitRepo(obj client.Object) []reconcile.Request {
 	ctx := context.Background()
 	var autoList imagev1.ImageUpdateAutomationList
-	if err := r.List(ctx, &autoList, client.InNamespace(obj.Meta.GetNamespace()), client.MatchingFields{repoRefKey: obj.Meta.GetName()}); err != nil {
-		r.Log.Error(err, "failed to list ImageUpdateAutomations for GitRepository", "name", types.NamespacedName{
-			Name:      obj.Meta.GetName(),
-			Namespace: obj.Meta.GetNamespace(),
-		})
+	if err := r.List(ctx, &autoList, client.InNamespace(obj.GetNamespace()),
+		client.MatchingFields{repoRefKey: obj.GetName()}); err != nil {
```

Comment:
The log message is removed here. I think it's ultimately OK (other GOTK controllers silently ignore errors in similar situations), I just want to flag it.